.github/workflows/codeql-analysis-linux.yml

```diff
@@ -0,0 +1,48 @@
+name: "CodeQL for Linux"
+
+on:
+  push:
+    branches: [ master ]
```

Comment:
So we only do scanning on master?

Reply:
Either on Push to master or on Pull Request to master.

I'm not sure what (other people's) forks will do once they update after this gets merged.  The documentation implies each repo needs to enable the scan -- but it doesn't specifically mention forks.